articles/machine-teaching-to-improve-ai/index.md

```diff
@@ -0,0 +1,131 @@
+
+![hero](/engineering-education/machine-teaching-to-improve-ai/hero.jpg)
+[Image Source](https://images.unsplash.com/photo-1587252229505-6c358e9009b5?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1051&q=80)
+
+We provide machine learning models with volumes of data with the expectation that they shall identify patterns or learn relationships in the data. This can be an expensive and time-consuming process. However, there exists an approach that leverages human abilities and expertise to guide machine learning models to find solutions to problems much faster. This is called machine teaching. We explore this approach in this article.
+
+### Table of Contents
+
+1. Machine teaching
+
+2. Machine teaching vs machine learning
+
+3. Benefits of machine teaching
+
+4. Applications of machine teaching
+
+### Prerequisites
+
+You are only required to be familiar with basic machine learning concepts. Here’s a link to an article that offers an introduction to [machine learning](/engineering-education/supervised-learning-algorithms/).
+
+### Machine Teaching
+
+Research around the field of machine learning today takes a technocentric approach. This approach is centered around technology and promotes its importance. It addresses technical and functional goals often at the expense of human values and ethics. As a result, this perspective may lead to some level of disconnection between the growth of artificial intelligence as a field and the users of the technology.
+
+To curb such disconnection, an alternative approach is emerging. This approach is placing the focus on domain experts and end-users as opposed to machine learning experts. The approach is referred to as machine teaching. The concept of machine teaching looks to obtain knowledge from people as opposed to data as the sole source. The idea behind machine teaching is adamant that any information processing skill that can be taught to a human can be easily taught to a machine. To come up with solutions to problems, machine teaching uses human expertise along with human abilities to assist machine learning models to identify crucial clues on how to generate a solution faster.
```

Comment:
Last sentence: To develop solutions to problems, machine teaching uses human expertise and human abilities to assist machine learning models in identifying crucial clues on how to generate a solution faster.